mappings/MGIAlzheimerMP_HPO_mapping.tsv

```diff
@@ -0,0 +1,23 @@
+subject_id	subject_label	predicate_id	object_id	object_label	match_type	sssom:confidence	sssom:creator_id	sssom:mapping_date	sssom:comment
```

Comment:
For both files, we need the official sssom header, like https://github.com/mapping-commons/mh_mapping_initiative/blob/master/mappings/mp_doid_pistoia_embedded.sssom.tsv#L1

in particular:

- license
- mapping_set_id
- curie_map